components/ui-api-layer/internal/domain/k8s/resourcequotastatus_resolver.go

```diff
@@ -0,0 +1,39 @@
+package k8s
+
+import (
+	"context"
+
+	"github.com/golang/glog"
+	"github.com/kyma-project/kyma/components/ui-api-layer/internal/domain/k8s/pretty"
+	"github.com/kyma-project/kyma/components/ui-api-layer/internal/gqlschema"
+	"github.com/kyma-project/kyma/components/ui-api-layer/pkg/gqlerror"
+	"github.com/pkg/errors"
+	"k8s.io/api/core/v1"
+)
+
+func newResourceQuotaStatusResolver(resourceQuotaStatusService *resourceQuotaStatusService) *resourceQuotaStatusResolver {
+	return &resourceQuotaStatusResolver{
+		statusSvc: resourceQuotaStatusService,
+	}
+}
+
+type resourceQuotaStatusResolver struct {
+	statusSvc *resourceQuotaStatusService
+}
+
+func (r *resourceQuotaStatusResolver) ResourceQuotasStatus(ctx context.Context, environment string) (gqlschema.ResourceQuotasStatus, error) {
+	resourcesToCheck := []v1.ResourceName{
```

Comment:
As discussed. IMO this is misleading because here is a slice where I can supposedly define what kind of resource should be taken into account when we are checking the resource quotas status. But when we will go to the service then we will discover that this is not true because some part of the calculation is hardcoded. My proposition is to remove from here this parametrization and move it directly to service. After that resolver will look like that:
```go
func (r *resourceQuotaStatusResolver) ResourceQuotasStatus(ctx context.Context, environment string) (gqlschema.ResourceQuotasStatus, error) {
	exceeded, err := r.statusSvc.CheckResourceQuotaStatus(environment)
	if err != nil {
		glog.Error(errors.Wrapf(err, "while getting %s [environment: %s]", pretty.ResourceQuotaStatus, environment))
		return gqlschema.ResourceQuotasStatus{}, gqlerror.New(err, pretty.ResourceQuotaStatus, gqlerror.WithEnvironment(environment))
	}

	return exceeded, nil
}
```